s3/libirods_s3.cpp

```diff
@@ -101,12 +102,12 @@
 // Pairing this with LIBS3 error injection will exercise the error recovery
 // and retry code paths.
 static boost::mutex g_error_mutex;
-static long g_werr{0}; // counter
-static long g_rerr{0}; // counter
-static long g_merr{0}; // counter
-static const long g_werr_idx{4}; // Which # pwrite to fail
-static const long g_rerr_idx{4}; // Which # pread to fail
-static const long g_merr_idx{4}; // Which part of Multipart Finish XML to fail
+static int64_t g_werr{0}; // counter
```

Comment:
Consider adding the `std::` prefix on the fixed-wdith integer types.

Although they aren't needed, including the namespace prefix future proofs us from changes to the C++ standard.

Reply:
dibe